src/application/agora/widgets/proposal.tsx

```diff
@@ -0,0 +1,103 @@
+import { classes } from 'shared/ui/utils';
+import { Button, Chip, WidgetBase } from 'shared/ui/components';
+
+import { ProposalData } from '../types';
+import { getDaysUntil, getEndsInLabel, getProposalUrl } from '../utils';
+
+interface Properties {
+  data: ProposalData;
+  className?: string;
+  top?: number;
+  isPreviousProposalAvailable: boolean;
+  isNextProposalAvailable: boolean;
+  loadingNextProposal: boolean;
+  onClose?: () => void;
+  showPreviousProposal: () => void;
+  showNextProposal: () => void;
+}
+
+export const Proposal = ({
+  data,
+  className,
+  top,
+  isPreviousProposalAvailable,
+  isNextProposalAvailable,
+  loadingNextProposal,
+  onClose,
+  showPreviousProposal,
+  showNextProposal,
+}: Properties) => {
+  const proposalEndDateInMs = new Date(
+    data.proposalData.endTimestamp,
+  ).getTime();
+
+  const navigationButtonClassName =
+    'inline-flex h-7 select-none items-center justify-center rounded-md text-sm font-light text-[#444444] transition-all hover:text-[#7d848a] focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-offset-2 disabled:pointer-events-none disabled:bg-white';
```

Comment:
I avoid storing tailwind classNames in variables because it usually breaks tailwind ecosystem tools like autocomplete etc. I'd suggest to move the button itself to a separate component in `agora/components` and just inline the classname there in the html